src/transaction/boot_sr.c

```diff
@@ -4253,7 +4253,7 @@ xboot_copy (REFPTR (THREAD_ENTRY, thread_p), const char *from_dbname, const char
 	  else
 	    {
 	      er_set (ER_WARNING_SEVERITY, ARG_FILE_LINE, ER_BO_DIRECTORY_DOESNOT_EXIST, 1, p);
-	      if (mkdir (p, 0777) < 0)
+	      if (mkdir (p, 0700) < 0)
```

Comment:
Don't we need to consider umask(S_IRGRP | S_IWGRP | S_IROTH | S_IWOTH)?

Reply:
If you as me, yes.
But it's not done using `umask` anywhere around the code.
Maybe a CBRD issue should be raised for this.

Reply:
Not sure.
Maybe.
If so, should this be reported as an upstream issue in `develop`?